ger_deps

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+cd
+sudo rm -rf /tmp/installitrude/
+mkdir /tmp/installitrude
+cd /tmp/installitrude
+clear
+
+# INSTALLATION OF LUAROCKS
+wget https://luarocks.org/releases/luarocks-3.9.2.tar.gz
+tar zxpf luarocks-3.9.2.tar.gz
+cd luarocks-3.9.2
+./configure && make && sudo make install
+echo -e "\n\e[32mLuarocks Installed successfully\e[0m\n"
+
+# INSTALLATION OF LUAFILESYSTEM
+sudo luarocks install luafilesystem
+echo "require \"lfs\"" | lua
```

Comment:
that good but we should probably actually check if it did the thing before saying installed successfully

Reply:
that could be done later on though, not of utmost priority, that's an other reason why we shouldn't clear, in case there was an error the user needs to figure out

Reply:
There is at the top of the file a line that says `set -e` which stops the execution as soon as an error is encountered. with this, if the installation or the require fails, the execution will stop

Reply:
ok, but we still shouldn't though

Reply:
I'll check